administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss

```
@@ -138,7 +138,9 @@ joomla-tab {
         background-color: var(--atum-bg-dark-60);
         background-image: none;
         border-right: 0;
-        box-shadow: none;
+        border-top: 1px solid transparent;
+        border-bottom: 1px solid #dee2e6;
+        box-shadow: none;       
```

```suggestion
        box-shadow: none;
```
Now the linter complained about unexpected white space at the end of this line.